lib/kv-redis.js

```diff
@@ -65,10 +106,33 @@ RedisKeyValueConnector.prototype.execute = function(command, args, cb) {
   this._client.sendCommand(cmd);
 };
 
+/**
+ * Disconnect from the data source.
+ *
+ * @callback {Function} callback
+ * @param {Error} err Error object.
+ *
+ * @header RedisKeyValueConnector.disconnect(cb)
+ */
 RedisKeyValueConnector.prototype.disconnect = function(cb) {
   this._client.quit(cb);
 };
 
+/**
+ * Persist a value and associate it with the given key.
+ *
+ * @param {String} modelName The model name to associate with the given key.
+ * @param {String} key Key to associate with the given value.
+ * @param {*} value Value to persist.
+ * @options {Number|Object} options Optional settings for the key-value
+ *   pair. If a Number is provided, it is set as the TTL (time to live) in ms
+ *   (milliseconds) for the key-value pair.
```

Comment:
> If a Number is provided, it is set as the TTL (time to live) in ms (milliseconds) for the key-value pair.

This is not true, connector methods do not support `options` as `Number`. This part is handled by KVAO so that we don't have to implement it in each connector.